<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>463</width>
    <height>641</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="maximumSize">
   <size>
    <width>463</width>
    <height>641</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Associate Setup</string>
  </property>
  <property name="styleSheet">
   <string notr="true">/* Set the color of text in PushButton to white */
QPushButton {
    background-color: #182454;
    color: white;
    border-radius: 4px;
    padding: 4px;
    font: 12pt &quot;Arial&quot;;
    height: 35px;
    
    
}
QPushButton:disabled{
    background-color: rgb(255, 225, 188);
    color: white;
    border-radius: 4px;
    padding: 4px;
    font: 12pt &quot;Arial&quot;;
    height: 35px;
}

/* Add a hover effect to buttons */
QPushButton:hover {
    background: #ff9d00;
    color: #ffffff
}

/* Add a pressed effect to buttons */
QPushButton:pressed {
    background: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
    stop: 0 #ff9d00, stop: 1 #ffb900);
}

QLabel{
	font: 87 9pt &quot;Arial&quot;;
   
	color: #182454;
    
}

QLineEdit{
    margin-bottom: 10px;
    border: 1px solid lightblue;
    
    height: 25px;
}
QLineEdit:focus{
    margin-bottom: 10px;
    border: 1px solid orange;   
}

QTimeEdit{
	border: 1px solid lightblue;
    font: 87 9pt &quot;Arial&quot;;
    height: 25px;
}
QTimeEdit:focus{
	border: 1px solid orange;
    font: 87 9pt &quot;Arial&quot;;
}

QDateTimeEdit{
	border: 1px solid lightblue;
    font: 87 9pt &quot;Arial&quot;;
    height: 25px;
}
QDateTimeEdit:focus{
	border: 1px solid orange;
    font: 87 9pt &quot;Arial&quot;;
}</string>
  </property>
  <property name="sizeGripEnabled">
   <bool>false</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_7">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="font">
        <font>
         <family>Arial</family>
         <pointsize>18</pointsize>
         <weight>50</weight>
         <italic>false</italic>
         <bold>false</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QLabel{
    color: #182454;
    font: 18pt &quot;Arial&quot;;
    font-weight: 400;
    margin-bottom: 10px;
}</string>
       </property>
       <property name="text">
        <string>Client Catcher</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_10">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="styleSheet">
        <string notr="true">QLabel{
    color: #182454;
    font: 8pt &quot;Arial&quot;;
    font-weight: 600;
}</string>
       </property>
       <property name="text">
        <string>Enter Your Information Below</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_2">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>*First Name:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="nameEdit">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="placeholderText">
        <string>First Name</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_3">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>*Last Name:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lastEdit">
       <property name="placeholderText">
        <string>Last Name</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_4">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>*Scheduling Link:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="schedulingEdit">
       <property name="styleSheet">
        <string notr="true">margin: 0px</string>
       </property>
       <property name="placeholderText">
        <string>Scheduling Link</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_2">
       <property name="styleSheet">
        <string notr="true">height: 25px;</string>
       </property>
       <property name="text">
        <string>Submit</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="Line" name="line">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_5">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="styleSheet">
        <string notr="true">QLabel{
    color: #ff9d00;
    font: 12pt &quot;Arial&quot;;
    font-weight: 600;
}</string>
       </property>
       <property name="text">
        <string>Optional Settings</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_6">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="styleSheet">
        <string notr="true">QLabel{
    color: #182454;
    font: 8pt &quot;Arial&quot;;
    font-weight: bold;
}</string>
       </property>
       <property name="text">
        <string>(These will help improve time suggestions)</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_8">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Lunch Time:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QTimeEdit" name="timeEdit">
       <property name="styleSheet">
        <string notr="true">margin-bottom: 8px</string>
       </property>
       <property name="time">
        <time>
         <hour>12</hour>
         <minute>0</minute>
         <second>0</second>
        </time>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_9">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Team Meeting (Next Occurence):</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDateTimeEdit" name="dateTimeEdit">
       <property name="date">
        <date>
         <year>2023</year>
         <month>1</month>
         <day>1</day>
        </date>
       </property>
       <property name="time">
        <time>
         <hour>8</hour>
         <minute>0</minute>
         <second>0</second>
        </time>
       </property>
       <property name="minimumDateTime">
        <datetime>
         <hour>9</hour>
         <minute>0</minute>
         <second>0</second>
         <year>1752</year>
         <month>9</month>
         <day>14</day>
        </datetime>
       </property>
      </widget>
     </item>
     <item alignment="Qt::AlignHCenter">
      <widget class="QCheckBox" name="checkBox">
       <property name="text">
        <string>Save Optional Settings</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="Line" name="line_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton">
       <property name="text">
        <string>Get Started</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../../Desktop/test.qrc"/>
 </resources>
 <connections/>
</ui>
